lib/tasks/represent_downstream.rake

```diff
@@ -10,7 +10,7 @@ namespace :represent_downstream do
   desc "Represent all editions downstream"
   task all: :environment do
     represent_downstream(
-      Edition.with_document.where.not(content_store: nil)
+      Document.joins(:editions).where(editions: { content_store: nil })
```

Comment:
This has lost the `not`.

My understanding is that this `where` is just an optimisation and there will always be one edition with a value of `draft` and/or one with `live`, and anything with a nil value is in a superseded state. I could be wrong though - I'm not sure why we have this here and not in the other tasks.